analytics.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/proto/livekit";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+service AnalyticsRecorderService {
+  rpc IngestStats(stream  AnalyticsStats) returns (google.protobuf.Empty){};
+  rpc IngestEvents(stream AnalyticsEvents) returns (google.protobuf.Empty){};
+}
+
+enum STATSSTREAMTYPE {
+  STATSUPSTREAM = 0;
```

Comment:
We don't need the prefixes here. Should we use upstream/downstream or publish/subscribe?

Reply:
will use Publish/Subscribe